src/app.ts

```diff
@@ -94,40 +94,39 @@ async function app() {
 
   fs.ensureDirSync(program.out);
 
-  const resourceDir = path.join(program.out, 'input', 'resources');
-  fs.ensureDirSync(resourceDir);
-
   fs.writeFileSync(
     path.join(program.out, 'package.json'),
     JSON.stringify(outPackage.config, null, 2),
     'utf8'
   );
 
+  logger.info('Exporting FHIR resources as JSON...');
```

Comment:
Now we always write the JSON files here.  The IG Exporter is no longer responsible for doing that separately.